azafea/event_processors/endless/metrics/utils.py

```diff
@@ -142,3 +142,7 @@ def get_event_datetime(request_absolute_timestamp: int, request_relative_timesta
     event_absolute_timestamp_sec = event_absolute_timestamp / 1000000000
 
     return datetime.fromtimestamp(event_absolute_timestamp_sec, tz=timezone.utc)
+
+
+def clamp_to_int64(u64: int) -> int:
```

Comment:
IIRC, currently there aren't cases where the values are needed/ more than 64-bit. However, I feel that some logging on those cases would be nice. I'm not sure what or how, but I've got a feeling it can help debugging further down the line. I don't see this issue as a blocker so I'm happy with creating a task to handle this later and proceed with this PR.

Reply:
> I feel that some logging on those cases would be nice. I'm not sure what or how, but I've got a feeling it can help debugging further down the line.

I agree, I can try to find a simple way to log this.

Reply:
I’ve added an error, as it’s done elsewhere in azafea. I’m not sure that it’s really useful, but it will be at least displayed when events are replayed, and that’s where most of the problems should be caught.